skimage/color/colorconv.py

```diff
@@ -1681,16 +1683,21 @@ def lab2lch(lab, *, channel_axis=-1):
     Returns
     -------
     out : (..., 3, ...) ndarray
-        The image in LCH format, in a N-D array with same shape as input `lab`.
+        The image in CIE-LCh color space, of same shape as input.
 
     Raises
     ------
     ValueError
-        If `lch` does not have at least 3 color channels (i.e. l, a, b).
+        If `lab` does not have at least 3 channels (i.e., L*, a*, and b*).
 
     Notes
     -----
-    The Hue is expressed as an angle between ``(0, 2*pi)``
+    The h channel (i.e., hue) is expressed as an angle in range ``(0, 2*pi)``.
+
+    References
+    ----------
+    .. [1] http://www.easyrgb.com/index.php?X=MATH&H=07
+    .. [2] https://en.wikipedia.org/wiki/HCL_color_space
```

Comment:
```suggestion
    .. [2] https://en.wikipedia.org/wiki/HCL_color_space
    .. [3] https://en.wikipedia.org/wiki/CIELAB_color_space
```
Doesn't hurt adding the CIE-LAB source here to?

Reply:
Am I missing something? I agree with not documenting the output range, however, this function involves the CIE-LAB color space as input. So I would definitely expect a linked reference in its docstring?

Reply:
Oops! CIE-LAB is the source color space and CIE-LCh the destination, so you're absolutely right. I was already thinking about `lch2lab` -- will take care of its docstring in an upcoming PR.